Tools/llm-tool/LLMTool.swift

```diff
@@ -203,29 +206,88 @@ struct EvaluateCommand: AsyncParsableCommand {
 
     @MainActor
     mutating func run() async throws {
-        let (modelContainer, modelConfiguration) = try await memory.start(args.load)
+        let modelContainer = try await memory.start { [args] in
```

Comment:
Can we rename this to `LMCommand` and subcommand `lm`, to match the `VLMCommand`. 

Alternatively (given the complexity) it might be worth using the same subcommand and just dispatching to the vlm subrouting if an image input is provided or not..

Reply:
Interesting idea!  The default model is different, as is the model factory.  We could certainly switch on the presence of an image (or video) to chose but I wonder if that complicates things over just having the two subcommands?

Let me try the refactor to fold these down into one and see if that looks reasonable.

- [ ] try refactor of vlm -> eval (lm) command

Reply:
I think that worked well -- it came down to this (mostly):

```swift
        // switch between LLM and VLM
        let vlm = image.count > 0
        if vlm {
            modelFactory = VLMModelFactory.shared
            defaultModel = MLXVLM.ModelRegistry.paligemma3bMix448_8bit
        } else {
            modelFactory = LLMModelFactory.shared
            defaultModel = MLXLLM.ModelRegistry.mistral7B4bit
        }
```